src/components/paragraphs/sum-top-banner/sum-top-banner-paragraph.tsx

```diff
@@ -17,47 +17,61 @@ type Props = HtmlHTMLAttributes<HTMLDivElement> & {
 
 const SumTopBannerParagraph = ({paragraph, pageTitle, ...props}: Props) => {
   const behaviors = getParagraphBehaviors(paragraph)
+  const bgColor = behaviors.sum_banner_behaviors?.sum_banner_overlay_bkg || "olive"
```

Comment:
This doesn't work. It should be `behaviors?.sum_top_banner_behavior?.sum_top_banner_overlay_bkg`

Reply:
Default to poppy light, not olive.